code-challenges.test.js

```diff
@@ -15,43 +15,144 @@
 // a) Create a test with an expect statement using the variable provided.
 // HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
 
-const colors1 = ["purple", "blue", "green", "yellow", "pink"]
+// describe("shuffleContent", () => {
+//   it("removes the first item from the array and shuffles the remaining content", () => {
+//     const colors1 = ["purple", "blue", "green", "yellow", "pink"]
+//     const colors2 = [
+//       "chartreuse",
+//       "indigo",
+//       "periwinkle",
+//       "ochre",
+//       "aquamarine",
+//       "saffron"
+//     ]
+//     expect(shuffleContent(colors1)).not.toEqual(expect.arrayContaining(expected))
+//     expect(shuffleContent(colors2)).not.toEqual(expect.arrayContaining(expected))
+//   })
+// })
+
+// const colors1 = ["purple", "blue", "green", "yellow", "pink"]
 // Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
-const colors2 = [
-  "chartreuse",
-  "indigo",
-  "periwinkle",
-  "ochre",
-  "aquamarine",
-  "saffron"
-]
+// const colors2 = [
+//   "chartreuse",
+//   "indigo",
+//   "periwinkle",
+//   "ochre",
+//   "aquamarine",
+//   "saffron"
+// ]
 // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+// FAIL  ./code-challenges.test.js
+// shuffleContent
+//   ✕ removes the first item from the array and shuffles the remaining content
+// ● shuffleContent › removes the first item from the array and shuffles the remaining content
+//   ReferenceError: shuffleContent is not defined
+
+// Pseudo code:
+// input: array
+// output: array with first item removed, then shuffles remaining items
+// process: 
+// iteration is needed, maybe for loop? or maybe .filter() ?
+// first remove first item of array, can use .shift() as it removes value at the first index.
+// Math.random() arrays index and not the value.
+
 
 // b) Create the function that makes the test pass.
 
-// Pseudo code:
+// const shuffleContent = (array) => {
+//   let newArr = []
+//   for ( let i=0; i < array.length; i++){
+//     newArr.shift(array[i].Math.random())
+//   }
+//   return newArr
+// }
+
+// console.log(shuffleContent(colors1, colors2))
 
 // --------------------2) Create a function that takes in an object that contains up votes and down votes and returns the end tally.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const votes1 = { upVotes: 13, downVotes: 2 }
+// describe("endTally", () => {
+//   it("takes in an object that contains up votes and down votes and returns the end tally", () => {
+//     const votes1 = { upVotes: 13, downVotes: 2 }
+//     const votes2 = { upVotes: 2, downVotes: 33 }
+//     expect(endTally(votes1)).toEqual("11")
+//     expect(endTally(votes2)).toEqual("-31")
```

Comment:
The function is returning a number, but you are telling the test to expect a string